test/e2e/atlas/streams_test.go

```diff
@@ -220,6 +219,32 @@ func TestStreams(t *testing.T) {
 		a.Equal("US_EAST_2", *privateLinkEndpoint.Region)
 		a.Equal("/subscriptions/fd01adff-b37e-4693-8497-83ecf183a145/resourceGroups/test-rg/providers/Microsoft.EventHub/namespaces/test-namespace", *privateLinkEndpoint.ServiceEndpointId)
 		a.Equal("test-namespace.servicebus.windows.net", *privateLinkEndpoint.DnsDomain)
+		a.Equal("Azure", privateLinkEndpoint.GetProvider())
+		a.Equal("US_EAST_2", privateLinkEndpoint.GetRegion())
+		a.Equal("/subscriptions/fd01adff-b37e-4693-8497-83ecf183a145/resourceGroups/test-rg/providers/Microsoft.EventHub/namespaces/test-namespace", privateLinkEndpoint.GetServiceEndpointId())
+		a.Equal("test-namespace.servicebus.windows.net", privateLinkEndpoint.GetDnsDomain())
+
+		// Assign the endpoint ID so that it can be used in subsequent tests
+		endpointID = privateLinkEndpoint.GetId()
+	})
```

Comment:
The changes above this were copied from https://github.com/mongodb/mongodb-atlas-cli/pull/3641. They're in a separate commit and will be reverted once the other PR is merged.